src/main/java/com/blackduck/integration/jenkins/polaris/service/PolarisCliVersionHandler.java

```diff
@@ -0,0 +1,55 @@
+package com.blackduck.integration.jenkins.polaris.service;
+
+import com.fasterxml.jackson.core.Version;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PolarisCliVersionHandler {
+    private static final String POLARIS_VERSION_PATTERN =
+            "Polaris Software Integrity Platform CLI Scan Client version - (\\d+\\.\\d+\\.\\d+)";
```

Comment:
@zaman-akib  Can we update the part `Polaris Software Integrity Platform CLI Scan Client version` of the patter `POLARIS_VERSION_PATTERN` to regex to find out the string? This will help us avoid breaking the plugin in case CLI updates this log ever in future.

Reply:
How about a regex for the following format: `[INFO] [one word] <one line> CLI Scan Client version - <version>\n`

I'm just trying to avoid the situation when we rebrand the log from `Polaris Software Integrity Platform` to `Coverity on Polaris Platform` :)

Reply:
@blackduck-saraf Updated the regex pattern

Reply:
Okay, the new regex addresses the concern and should be good.